15special.tex

```diff
@@ -150,25 +150,25 @@
 \subsection[Jitter]{\protect\hyperlink{Jitter}{Jitter}}
 The following steps are now performed to determine the jittered starting parameter values (illustrated in Figure \ref{fig:jitter}):
 \begin{enumerate}
-	\item A normal distribution is calculated such that the pr(P\textsubscript{MIN}) = 0.1\% and the pr(P\textsubscript{MAX}) = 99.9\%.
-	\item A jitter shift value, termed ``\textit{K}'', is calculated from the distribution equal to pr(P\textsubscript{CURRENT}).
-	\item A random value is drawn, ``\textit{J}'', from the range of \textit{K}-jitter to \textit{K}+jitter.
+	\item A normal distribution is calculated such that the $pr(P_{MIN}) = 0.1\%$ and the $pr(P_{MAX}) = 99.9\%$.
+	\item A jitter shift value, termed ``$K$'', is calculated from the distribution equal to $pr(P_{CURRENT})$.
+	\item A random value is drawn, ``$J$'', from the range of $K$-jitter to $K$+jitter.
 	\item Any value which falls outside the 0-1 range (in the cumulative normal space) is mapped back from the bound to a point one-tenth of the way from the bound to the initial value.
-	\item \textit{J} is a new cumulative normal probability value.
-	\item Calculate a new parameter value, P\textsubscript{JITTERED}, such that pr(P\textsubscript{JITTERED}) = \textit{J}.
+	\item $J$ is a new cumulative normal probability value.
+	\item Calculate a new parameter value, $P_{JITTERED}$, such that $pr(P_{JITTERED}) = J$.
 \end{enumerate}
 
 \begin{figure}[ht]
 	\begin{center}
-		\includegraphics[alt={Illustration of the jitter algorithm.},scale = 0.75]{jitter_illustration}\\
+		\includegraphics[alt={Plot showing parameter space on the x-axis along and transformed space on the y-axis. A cumulative normal line is shown where the 0.001 and 0.999 quantiles are set to min and max respectively. A vertical stack of horizontal bars show the distribution of transformed initial values plus U. The distribution is shown on the parameter space axis with the initial input value in gray and the new init in red. Red arrows on the cumulative normal line show the random U written a negative jitter value comma positive jitter value.},scale = 0.75]{jitter_illustration}\\
```

Comment:
I think this alt text can be fine enough for now.  However, I think we may want to consider whether the description provided gives the reader the main takeaway from the figure.